src/obu.c

```diff
@@ -255,6 +259,7 @@ static avifBool parseSequenceHeaderEnabledFeatures(avifBits * bits, avifSequence
     return !bits->error;
 }
 
+// Note: Does not parse separate_uv_delta_q.
```

Comment:
It may be better to fork this function into an AV1 and an AV2 version because I am planning to change the `chroma_sample_position` syntax element in AV2. I will try that in a follow-up pull request.